torchvision/prototype/transforms/_container.py

```diff
@@ -25,38 +25,38 @@ def __init__(self, transform: Transform, *, p: float = 0.5) -> None:
         self.transform = transform
         self.p = p
 
-    def forward(self, *inputs: Any, params: Optional[Dict[str, Any]] = None) -> Any:
+    def forward(self, *inputs: Any) -> Any:
```

Comment:
Question: I assume this is removed for the same reasons `Compose` doesn't support it. Aka it's hard to pass parameters when you have many transforms. Right?

Reply:
That is one reason. Plus, the previous design handled the complete parameter passing through the `get_params` method. Since that is now private and only handles the dynamic part of the parameters, accepting a `params` argument for the call opens up more problems than it solves.

Reply:
I spoke with @NicolasHug about `get_params()` being private and he mentioned a couple of valid use cases on why we might want to keep it public. We can review this later but I agree that removing it from here is probably for the best.